src/flows/components/Sidebar.tsx

```diff
@@ -214,6 +214,13 @@ const Sidebar: FC = () => {
           title: 'Export to Client Library',
           menu: <ClientList />,
         },
+        {
+          title: 'Preview Subsequent',
```

Comment:
why is this in here?

Reply:
more constructively, the intent of this product change is to make iterating on an individual panel feel faster. less waiting for irrelevant panels to run, faster iteration per panel (submitting multiple times before discovering the right flux combination), lighter weight exploration. hiding that behind a dropdown and giving a weird name like "preview subsequent" seems counter productive? can we instead put it in the footer of every panel with a play icon that says "preview / run from here"?

Reply:
Sure thing. This was an initial pass at putting this somewhere, precisely so i could get this feedback. Your point makes perfect sense, ill drop it in the footer! Thanks @drdelambre